src/test/java/org/jabref/model/event/TestEventListener.java

```diff
@@ -25,9 +25,10 @@ public void listen(AllInsertsFinishedEvent event) {
         this.firstInsertedEntry = event.getBibEntry();
     }
 
+    // Not sure if this will work
```

Comment:
Looks like it works.

Reply:
I just made some changes to adapt the event to multiple entries.